docs/topics/testing/tools.txt

```diff
@@ -1661,8 +1661,7 @@ your test suite.
     of ``needle`` occurrences will be strictly verified.
 
     Whitespace in most cases is ignored, and attribute ordering is not
-    significant. The passed-in arguments must be valid HTML. See
```

Comment:
We can remove it because `assertInHTML()` docs contains a note: _"Asserts that **the HTML fragment** ``needle`` is contained in ..."_